packages/xo-web/src/common/xo/index.js

```diff
@@ -1787,7 +1787,7 @@ export const reconfigurePifIp = (pif, { mode, ip, netmask, gateway, dns }) =>
     netmask,
     gateway,
     dns,
-  })
+  }).then(() => (pif.management ? _call('pif.managementReconfigure', { pif: resolveId(pif) }) : noop))
```

Comment:
Why is it not done automatically in `pif.reconfigureIp`?

Reply:
It have to be done only in case of `management network` no ?